recipes/dbus/1.x.x/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+    "1.12.20":
```

Comment:
`1.12.20` is from July 2020.
`1.13.18` is available too.
Would that be a valid version too?